website/src/menu.css.ts

```diff
@@ -73,6 +81,11 @@ export const navButton = style([
   },
 ])
 
+export const navButtonChoices = styleVariants({
+  cwkw: [navButton, { color: cwkwTheme.colors.text }],
```

Comment:
So here, I wouldn't make variants. Instead change the `navButton` styles so that switching themes does this for us.